web/beacon-app/src/utils/formatDate.ts

```diff
@@ -1,14 +1,27 @@
 export const isDate = (value: any) => value instanceof Date;
 
-export const formatDate = (date: Date, format?: string) => {
+export const formatDate = (date: Date, format?: string, type?: string) => {
   if (!format) {
```

Comment:
After making the change, the date format didn't change. They're returned as strings, so that's possibly why. Removing `hour` and `minute` here changed the date's format. However, I kept the addition that was suggested, so please let me know how you'd like to proceed.